core/chains/evm/chain_set.go

```diff
@@ -73,7 +73,7 @@ func (cll *chainSet) Start(ctx context.Context) error {
 	var ms services.MultiStart
 	for _, c := range cll.Chains() {
 		if err := ms.Start(ctx, c); err != nil {
-			return errors.Wrapf(err, "failed to start chain %q", c.ID())
+			return errors.Wrapf(err, "failed to start chain %q", c.ID().String())
```

Comment:
Based on the history, we should also change %q to %s.

https://github.com/smartcontractkit/chainlink/commit/7b3a1201bb087cd6fedb9baa86407d7e22f2afd4#diff-39bc8851b241cc41173ad6d1848c2ab70bed8dc16d30352f25ae7cf9bfe139c9

Reply:
What do you mean? The %q is deliberate to quote the value, and compatible with `string`. It was only the `big.Int`s `Format` method getting in the way.